docs/installation.md

```diff
@@ -106,6 +110,25 @@ Add to `ios/_YOUR_PROJECT_NAME_/AppDelegate.m:
 
 This should be the **first line** of the method.
 
+#### If you are not using CocoaPods
+
+If you installed via `react-native-link`, add the following to your
+`package.json` and replace the `REPLACE_ME_RELATIVE_PATH_TO_GOOGLE_MAPS_INSTALL`
+with the relative path from your project root to the directory in which you
+installed the Google Maps frameworks:
+
+```json
+{
+  "name": "your-app",
+  "scripts": {
+    "postinstall": "./node_modules/react-native-maps/enable-google-maps REPLACE_ME_RELATIVE_PATH_TO_GOOGLE_MAPS_INSTALL"
+  }
```

Comment:
we may add this to lib if we define a standard method of installation @rborn

Reply:
@alvelig @wbyoung I think we need to educate the user how to install the google maps sdk (I have no idea for example right now at 1:45 AM and some 🍻)

Reply:
@wbyoung can you help a bit with @rborn suggestion, I wasn't able to do it myself in my own project (only inside a clean repo) so I'm not sure am I doing it right

Reply:
I don't understand how to get `REPLACE_ME_RELATIVE_PATH_TO_GOOGLE_MAPS_INSTALL`. Does someone have more info about this? Is it to our `ios` folder?

Reply:
@annelorraineuy I have the google maps sdk downloaded to `/Work/sdks/GoogleMaps-2.7.0/` so you need to put that path instead of the `REPLACE_ME_RELATIVE_PATH_TO_GOOGLE_MAPS_INSTALL`

Reply:
@annelorraineuy if you got it working & have documentation improvements, please send a PR. I followed the style of various documentation sources that I've used over time to try to make it clear and succinct, but it's always good to try to improve for everyone!